megamek/src/megamek/client/ui/IBoardView.java

```diff
@@ -161,9 +161,11 @@ public void setMovementEnvelope(Map<Coords,Integer> mvEnvData,
      * mapsheets), draws the hexes onto it, and returns that image.
      *
      * @param ignoreUnits If true, no units are drawn, only the board
+     * @param useBaseZoom If true, save the image at zoom = 1, otherwise at the
+     * current board zoom.
      * @return
```

Comment:
Wait, isn't this the exact opposite of what it does?

Reply:
I don't think so.

```
if (useBaseZoom) {
    zoomIndex = BASE_ZOOM_INDEX;
    zoom();
}
```

Reply:
Just tested it again and it seems to do what I want it to do, manual image saves work at the current board zoom.